tests/test_indices.py

```diff
@@ -95,3 +97,48 @@ def test_NAO_naming(source_name):
     NAO = north_atlantic_oscillation(data_set)
 
     assert NAO.name == "NAO"
+
+@pytest.mark.parametrize("source_name", list(VARNAME_MAPPING.keys()))
+def test_enso34_zeromean(source_name, rtol = 1e-8):
```

Comment:
I'm a little torn between liking and disliking the logic here: On one hand, we should not have implicit assumptions about the objects we are testing without asserting them. On the other hand, we test something wich we constructed to be of order 1 (a standardized index). So I'd personally just go for something like
```python
def test_enso_mean_is_zero():
    # ... calculate enso
    scale = enso.std("time")
    n = len(enso)
    bound = 2 * (scale / n ** 0.5)
    assert abs(enso.mean("time")) < bound
```